lib/tasks/add_badges_for_beginner_trainings.rake

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+namespace :add_badges_to_beginner_trainings do
+  desc "Add badges to beginner trainings from September 1st 2022 to now"
+  task run: :environment do
+    training_sessions_ids =
+      TrainingSession.where(
+        created_at:
+          DateTime.new(2022, 9, 1).beginning_of_day..DateTime.now.end_of_day,
+        level: "Beginner"
+      ).pluck(:id)
+    updated_certifications = []
+    no_template_certifications = []
+    failed_certifications = []
+
+    Certification
+      .where(training_session_id: training_sessions_ids)
+      .each do |c|
+        badge_template = BadgeTemplate.find_by(training_id: c.training.id)
+
+        if badge_template.present?
+          updated_certifications << "#{c.user.name} - #{badge_template.badge_name} (#{b.acclaim_badge_id})"
```

Comment:
B isn't defined, since the same thing is pushed into updated_certifications on 32 I assume we can get rid of this?